lpc845/rust-lpc845.sublime-project

```diff
@@ -0,0 +1,11 @@
+{
+	"folders":
+	[
+		{
+			"path": ".",
+			"file_exclude_patterns": [
+				"target/**"
+			]
+		}
+	]
+}
```

Comment:
Just a note, the LPC82x Sublime project file was just there for my own convenience, so don't feel the need to add it unless you're also using Sublime Text.